### PR TITLE
Add documentation about OBS builds

### LINKED ISSUE #2202: Documentation for the OBS builds is missing
> estimate 1

- Related to #2188 
- The modification is to user documentation
- The change is an addition
- The reason for the change is we now support OBS builds of dev and release-v2.0-candidate branches through OBS
- The documentation component to change is to add URLs to main README in web pages section
- Also change the https://geopm.github.io/build.html documentation because non-developers should just use the OBS packages

### PR DESCRIPTION
**Pull request by cmcantalupo**
- Fixes #2202

### CODE REVIEW

File: service/docs/Makefile.mk [hunk 1]
```diff
@@ -48,6 +48,7 @@ EXTRA_DIST += docs/geninfo.sh \
               docs/source/controls_SKX.rst \
               docs/source/devel.rst \
               docs/source/index.rst \
+              docs/source/istall.rst \
```
Comment (bgeltz) [inline service/docs/Makefile.mk]:
```suggestion
              docs/source/install.rst \
```

```test-dist``` is failing.

File: service/docs/source/install.rst [hunk 2]
```diff
@@ -0,0 +1,92 @@
+
+Install
+=======
+
+Continuous integration creates packages for a variety of Linux
+distributions each time certain branches hosted on Github are updated.
+This is supported by Github Actions, in conjunction with the OpenSUSE
```
Comment (bgeltz) [inline service/docs/source/install.rst]:
```suggestion
distributions each time certain branches hosted on GitHub are updated.
This is supported by GitHub Actions, in conjunction with the OpenSUSE
```

File: service/docs/source/install.rst [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+    https://download.opensuse.org/repositories/home:/geopm:/release-v2.0-candidate/
+
+The packages built from this service are named:
+
+- ``geopm-service``
+- ``geopm-service-devel``
+- ``libgeopmd0``
+- ``python3-geopmdpy``
+
```
Comment (bgeltz) [inline service/docs/source/install.rst:25]: What about dasbus?  Aren't we building some of our dependencies in OBS?
They should be listed here.

File: service/docs/source/install.rst [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+   + `release-v2.0-candidate <https://download.opensuse.org/repositories/home:/geopm:/release-v2.0-candidate/Fedora_35/home:geopm:release-v2.0-candidate.repo>`__
+- CentOS-8
+   + `dev <https://download.opensuse.org/repositories/home:/geopm/CentOS_8/home:geopm.repo>`__
+   + `release-v2.0-candidate <https://download.opensuse.org/repositories/home:/geopm:/release-v2.0-candidate/CentOS_8/home:geopm:release-v2.0-candidate.repo>`__
+- CentOS-8-Stream
+   + `dev <https://download.opensuse.org/repositories/home:/geopm/CentOS_8_Stream/home:geopm.repo>`__
+   + `release-v2.0-candidate <https://download.opensuse.org/repositories/home:/geopm:/release-v2.0-candidate/CentOS_8_Stream/home:geopm:release-v2.0-candidate.repo>`__
+
+One of these URLs should be added to the package configuration by
+running one of the two commands below:
```
Comment (bgeltz) [inline service/docs/source/install.rst]:
```suggestion
running one of the two commands below depending on the package management system for the operating system:
```

File: service/docs/source/install.rst [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+.. code-block:: bash
+
+    # On SUSE based distros
+    zypper addrepo <URL>
+
+    # On RH based distros
+    yum-config-manager --add-repo <URL>
+
+depending on the package management system for the operating system.
```
Comment (bgeltz) [inline service/docs/source/install.rst]:
```suggestion
```

File: service/docs/source/install.rst [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+Example Commands
+----------------
+
+As an example, the following two bash commands will add the
+development branch OBS build repository for OpenSUSE 15.3 and install
+the GEOPM Service.
+
+.. code-block:: bash
+
+   zypper addrepo https://download.opensuse.org/repositories/home:/geopm/
```
Comment (bgeltz) [inline service/docs/source/install.rst]:
```suggestion
   zypper addrepo https://download.opensuse.org/repositories/home:/geopm/15.3/
```

**Review by bgeltz — CHANGES_REQUESTED** [review]

**Review by bgeltz — APPROVED** [review]

**Comment by cmcantalupo** [comment]
@bensallen This documentation is now posted here:

https://geopm.github.io/install.html

**Pull request merged**